### PR TITLE
[fix] Fixed firmware private storage missing dir #270

### LINKED ISSUE #270: [bug] Private firmware image dir missing from volume mounts
The directory containing firmware images uploaded when firmware upgrader module is enabled is missing from the volume mount definition.

### PR DESCRIPTION
**Pull request by Aryamanz29**
Added the missing firmware image private storage directory to the volume mounts.

Screenshot from 2023-03-17 19-53-00

Screenshot from 2023-03-17 16-19-43

Fixes #270

### CODE REVIEW

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
Looking good, see my comment below.

File: deploy/examples/kubernetes/Deployment.yml [hunk 1]
```diff
@@ -114,6 +119,9 @@ spec:
             - name: openwisp-media-data
               mountPath: /opt/openwisp/public/media
               readOnly: true
+            - name: openwisp-private-storage-data
+              mountPath: /opt/openwisp/public/private
```
[the remark this answers is not in the document]
Comment (Aryamanz29) [reply deploy/examples/kubernetes/Deployment.yml:123]: Yeah, @nemesisdesign, so should I remove it? or rename it to something else?

[the remark this answers is not in the document]
Comment (nemesisdesign) [reply deploy/examples/kubernetes/Deployment.yml:123]: What about `/opt/openwisp/private`?

**Review by nemesisdesign — APPROVED** [review]

File: deploy/examples/kubernetes/Deployment.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Aryamanz29) [reply deploy/examples/kubernetes/Deployment.yml]: thanks @nemesisdesign.

**Pull request merged**